api/server.go

```diff
@@ -0,0 +1,38 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"go.uber.org/zap"
+)
+
+// Ensure that we implement the server interface
+var _ ServerInterface = (*Server)(nil)
+
+type Server struct{
+	logger *zap.SugaredLogger
+}
+
+func NewServer(logger *zap.SugaredLogger) *Server {
+	return &Server{
+		logger: logger,
+	}
+}
+
+func (s *Server) RatsdChares(w http.ResponseWriter, r *http.Request, param RatsdCharesParams) {
+	var requestData ChaResRequest
+
+	err := json.NewDecoder(r.Body).Decode(&requestData)
+	if err != nil {
+		s.logger.Error("fail to retrieve nonce from request")
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	s.logger.Info("request nonce: ", requestData.Nonce)
+	s.logger.Info("request media type: ", *(param.Accept))
+	w.Header().Set("Content-Type", "application/eat+jwt; eat_profile=\"tag:github.com,2024:veraison/ratsd\"")
+	w.WriteHeader(http.StatusOK)
+	w.Write([]byte("hello, rastd!"))
+}
```

Comment:
Question: do you want to add all the checks on input conformance (e.g., C-T, nonce format), and content negotiation in a subsequent PR along with the tests?

If so, I am OK with that, but we need to make that a bit more explicit - i.e., with one or more well-scoped tracking issue(s).

Reply:
@thomas-fossati Should I go ahead to create a Github issue to state that it warrants a format check? I'd want to make clear the format of C-T and nonce to proceed on.

Reply:
Please, go ahead and create the issue.  The things to check are, at least:
* payload consistency (`Content-Type` header and format of the JSON payload)
* HTTP content negotiation (obey any `Accept` header)